### PR TITLE
Run PSR2R.Commenting.FullyQualifiedClassNameInDocBlockSniff

### PR DESCRIPTION
**Pull request by dereuromark**
and some readme cleanup :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,15 +34,12 @@ Add the following to the ```Webservice``` section of your application config.
...
 ```php
-<?php
-
 namespace App\Controller;
 
 use Cake\Event\Event;
 
 class StatusesController extends AppController
 {
-
```
Comment (Marlinc) [inline README.md:45]: Classes should have a empty line at the start: https://github.com/cakephp/cakephp/blob/master/src/Controller/ErrorController.php#L27

File: README.md [hunk 2]
```diff
@@ -64,15 +61,12 @@ class StatusesController extends AppController
...
 ```php
-<?php
-
 namespace App\Shell;
 
 use Cake\Console\Shell;
 
 class StreamShell extends Shell
 {
-
```
Comment (Marlinc) [inline README.md:75]: Same here.

File: README.md (hunk 1 above)
Comment (dereuromark) [inline README.md:45]: I agree with you for PSR2R ;) But for PSR2 I think there is no such empty line.
See http://www.php-fig.org/psr/psr-2/

PS: You can always switch to the more sane PSR2R of course :)

Comment (Marlinc) [inline README.md:45]: I see, in that case you're right!

**Comment by Marlinc** [comment]
Thanks a lot!

**Pull request merged**